src/map/battle.c

```diff
@@ -8014,6 +8014,16 @@ static void battle_adjust_conf(void)
 	}
 #endif
 
+#if !(PACKETVER_MAIN_NUM >= 20160323 || (PACKETVER_RE_NUM >= 20160316 && defined(ENABLE_RODEX_ACCOUNT_MAIL_RE_PATCH)) || defined(PACKETVER_ZERO))
+	if (battle_config.feature_rodex_use_accountmail == 1) {
+		ShowWarning("conf/map/battle/feature.conf:features/rodex_use_accountmail RoDEX account mail is enabled but it requires 2016-03-16 RagexeRE / 2016-03-23 Ragexe or newer, disabling...\n");
```

Comment:
need disable this warning for ci, because it failing builds

Reply:
oh, oops. right.

I am wondering, should I disable it with `#ifndef BUILDBOT` or add another grep to travis.sh like you did for mysql?

Reply:
yes BUILDBOT should be fine

Reply:
hmm these ifs got a bit... messy, but I think it is fine since it is in a specific place. if you have any better suggestion let me know